index.bs

```diff
@@ -1810,7 +1809,71 @@ partial interface MLGraphBuilder {
 </div>
 
 <details open algorithm>
+  <summary>
+    To <dfn for=MLGraphBuilder>calculate conv2d padding</dfn> given unsigned integers |inputSize|, |filterSize|, |stride| and |dilation| and {{MLAutoPad}} |autoPad|, perform these steps. They return a [=/list=] of 2 numbers.
```

Comment:
@inexorabletash 

There is a proposal to drop the support of [`MLAutoPad`](https://www.w3.org/TR/webnn/#enumdef-mlautopad) and only support explicit padding. Should we coordinate with that proposal? For example, I can send a PR to remove `MLAutoPad`. What do you think?

@fdwr

Reply:
SGTM - it would definitely simplify this a bunch!

Reply:
Yes please.

Reply:
Here it was: https://github.com/webmachinelearning/webnn/issues/326

Reply:
https://github.com/webmachinelearning/webnn/pull/587